packages/__docs__/src/Hero/index.tsx

```diff
@@ -348,6 +348,7 @@ class Hero extends Component<HeroProps> {
             <Img
               src="https://instui-docs.s3.us-east-2.amazonaws.com/hero2.jpg"
               display="block"
+              rel="prefetch"
```

Comment:
This fixes the "The resource  was preloaded using link preload but not used within a few seconds" warnings.